hyppo/multivariate/tests/test_dhsic.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_almost_equal
+
+from ...tools import linear, power
+from .. import Dhsic
+
+
+class TestDhsicStat:
+    @pytest.mark.parametrize("n, obs_stat", [(100, 0.04561), (200, 0.03911)])
+    @pytest.mark.parametrize("obs_pvalue", [1 / 1000])
+    def test_linear_oned(self, n, obs_stat, obs_pvalue):
+        np.random.seed(123456789)
+        x, y = linear(n, 1)
+        stat, pvalue = Dhsic(**{'gamma': 0.5}).test(*(x, y))
```

Comment:
`*(x, y)` is redundant. You should be able to do `x, y`. Same goes with `**{'gamma': 0.5}`

Reply:
I was able to run it with x,y but it doesn't work with {'gamma':0.5} if I take out the **. Is there a different way to call **kwargs?

Reply:
gamma=0.5 should be sufficient